generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -107,14 +107,14 @@ enum OptName {
   OPT_NAME_IPV6_MULTICAST_IF = 14;
   OPT_NAME_IPV6_MULTICAST_HOPS = 15;
   OPT_NAME_IPV6_V6ONLY = 16;
-  OPT_NAME_SO_RX_DATA = 257;
```

Comment:
@reckenro and I have talked about this and it's in several places in this change: I'm ooching to follow the header naming more closely except to match strict naming conventions. So, in this case, don't add extra tokens to the ALLCAPS suffix, since that's valid as an enum value name.